components/compositing/compositor.rs

```diff
@@ -76,9 +76,9 @@ trait ConvertScrollRootIdFromWebRender {
     fn from_webrender(&self) -> ScrollRootId;
 }
 
-impl ConvertScrollRootIdFromWebRender for usize {
+impl ConvertScrollRootIdFromWebRender for u64 {
     fn from_webrender(&self) -> ScrollRootId {
-        ScrollRootId(*self)
+        ScrollRootId(*self as usize)
```

Comment:
The problem with this is that is lossy in 32-bit platforms... But probably that's ok assuming we only pass `usize`s to WR. Perhaps leave a comment or something?